src/functions/execute.js

```diff
@@ -43,14 +46,14 @@ module.exports = (message) => {
                     //Get welcome channel
                     message.reply(
                         "Channel currently is set to " +
-                            (await getGuild(message.guild.id).welcomeChannel)
+                            (getGuild2(message.guild).welcomeChannel)
```

Comment:
That will not work. `getGuild2` will be executed asynchronously and resultant string will be `Channel currently is set to undefined`

Reply:
Can you help me, I am not used to es6

Reply:
Okay. I'll make a commit